ssl/tls13_enc.c

```diff
@@ -46,14 +40,28 @@ int tls13_hkdf_expand(SSL *s, const EVP_MD *md, const unsigned char *secret,
      * prefix and label + bytes for the label itself + 1 byte length of hash
      * + bytes for the hash itself
      */
-    unsigned char hkdflabel[sizeof(uint16_t) + sizeof(uint8_t) +
-                            + sizeof(label_prefix) + TLS13_MAX_LABEL_LEN
-                            + EVP_MAX_MD_SIZE];
+    static unsigned char hkdflabel[sizeof(uint16_t) + sizeof(uint8_t) +
```

Comment:
I suppose static here is a no-go as it would make the function non-reentrant. So it either needs to be kept on the stack or allocated.